### PR TITLE
py3k

### LINKED ISSUE #4: Fix deprecation warnings

### PR DESCRIPTION
**Pull request by hexylena**
- fixes #4 
- virtualenv stuff that might not be necessary, from galaxy
- force `become: true` on systemd, not a good solution

with virtualenv/python3 installed, things seemed to work once I removed the stupid "install venv if missing" task I added a while back for unclear reasons. It's just a hard requirement that it be there.

### CODE REVIEW

**Comment by hexylena** [comment]
Ok @natefoo how's that look

File: README.md [hunk 1]
```diff
@@ -41,6 +44,11 @@ the `pip_virtualenv_command` variable.
 [venv]: http://virtualenv.readthedocs.org/
 [pip]: http://pip.readthedocs.org/
 
+Breaking Changes
+----------------
+
+As of 1.0.0, installation with git+pip, and from source, are no longer supported.
```
Comment (natefoo) [inline README.md]: > installation with git+pip ... no longer supported

I still need this. But I think it should be trivial to support if the argument to the `pip` module's `name` and `version` options are parameterized?

Comment (hexylena) [reply README.md]: Pushed bfd8883f9ff953bc231824431ac66b1d9cd7baf4 which should help

File: tasks/virtualenv.yml [hunk 2]
```diff
@@ -31,7 +31,8 @@
 
     - name: Install Pulsar
       pip:
-        name: pulsar-app
+        name: "{{ pulsar_package_name }}"
+        vesion: "{{ pulsar_package_version | default(omit) }}"
```
Comment (natefoo) [inline tasks/virtualenv.yml]:
```suggestion
        version: "{{ pulsar_package_version | default(omit) }}"
```

File: README.md (hunk 1 above)
Comment (natefoo) [reply README.md]:
```suggestion
As of 1.0.0, installation directly from source using `git clone` is no longer supported. However, `pulsar_package_name` and `pulsar_package_version` can be used to install using pip's `git+https` method.
```

**Review by natefoo — APPROVED** [review]

**Pull request merged**